src/data/NewsData.js

```diff
@@ -0,0 +1,116 @@
+export const NewsData = [
+  {
+    id: 0,
+    year:2024,
+    month: "01",
+    date: "07",
+  },
```

Comment:
```suggestion
  {
    id: 0,
    year:2024,
    month: "01",
    date: "07",
  },
```
여기서 year은 string이고, month는 number인 이유가있는지? 일반적으론 통일시켜주는게 맞음.
그리고 일자는 보통 date보단 day로 표현함.

Reply:
그리고 광고데이터면 광고에 대한 content도 있어야하지 않을까?

Reply:
광고 내용은 지금 하고 있는 부분 하고 진행할려고요!